### PR TITLE
added dependencies for clipmenu

### PR DESCRIPTION
**Pull request by dkwo**
I added some dependencies.
For example, without clipnotify I get an error.
These are the same dependencies as Arch Linux.

### CODE REVIEW

**Comment by ericonr** [comment]
Please follow commit format: `clipmenu: add runtime dependencies.` or similar.

File: srcpkgs/clipmenu/template [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Template file for 'clipmenu'
 pkgname=clipmenu
 version=6.1.0
-revision=1
+revision=2
 archs=noarch
```
Comment (ericonr) [inline srcpkgs/clipmenu/template:5]: Please remove noarch, as well.

**Comment by dkwo** [comment]
@ericonr I did as you suggested.

**Comment by ericonr** [comment]
Thanks!

**Pull request merged**